calico-cloud/get-started/install-automated.mdx

```diff
@@ -39,6 +42,110 @@ When the key is deleted, all API requests based on that key will be rejected.
 
 :::
 
+## About customizing your Helm installation
+
+<!-- As a user, I want to know what I can do with  so I can  -->
+
+You can customize your Calico CLoud installation for the following purposes:
+
+* to enable or disable certain features
+* to modify pod scheduling and resource management
+
+To do this, you can either edit the default `values.yaml` file or pass individual key-value pairs using the `--set` flag for the `helm upgrade` command.
+
+### Required parameters
+
+You need to set these.
+
+| Parameter | Value | Example | Description |
+| -- | -- | -- | -- |
+| `installer.clusterName` | string | `cluster-name` | The name given to your managed cluster in Calico Cloud. |
+| `installer.calicoCloudVersion` | string | `{{cloudUserVersion}}` | The version of Calico Cloud you're installing. |
+
+
+```yaml title="Example from values.yaml with clusterName and calicoCloudVersion"
+installer:
+  clusterName: example-cluster
+  calicoCloudVersion: {{cloudUserVersion}}
+```
+
+### Optional parameters for private registries
+
+These are for private registries.
+
+| Parameter | Value | Example | Description |
+| -- | -- | -- | -- |
+| `installer.registry` | string | `registry-name` | The name given to your managed cluster in Calico Cloud. |
+| `installer.imagePath` | string | `image-path` | The version of Calico Cloud you're installing. |
+| `imagePullSecrets.name` | string | `secret-name` | The version of Calico Cloud you're installing. |
+
+### Optional parameters for features
+
+These are for features. You can't change them in the UI if you're installing with Helm.
+
+| Feature name | Parameter | Values |
+|---------|-----|--------|
+| Image Assurance | `installer.components.imageAssurance.state` | `Enabled` (default), `Disabled` |
+| Container Threat Detection | `installer.components.runtimeSecurity.state` | `Enabled`, `Disabled` (default) |
+| Security Posture Dashboard | `installer.components.securityPosture.state` | `Enabled` (default), `Disabled` |
+| Packet Capture | `installer.components.packetCaptureAPI.state` |  `Enabled`, `Disabled` (default) |
+| Compliance Reports | `installer.components.compliance.enabled` | `true` (default), `false` |
+
+:::note
+
+If you're upgrading from an older version of Calico Cloud, the Container Threat Detection and Packet Capture features will remain enabled unless you explicitly set them to `Disabled`.
+
+:::
+
+### Optional parameters for pod scheduling and resource management
+
+For many Calico Cloud components, you can specify node selectors, tolerations, and resource requests and limits.
+The full list of Calico Cloud components is available in the default `values.yaml` file.
```

Comment:
Do we only want to provide this info in the values.yaml file? There is plenty of reference documentation we could link to by including things like
`installer.components.apiServer` and linking to the reference/installation/api#operator.tigera.io/v1.APIServerDeployment